### PR TITLE
Fix316

### PR DESCRIPTION
**Pull request by jw3126**
#316

### CODE REVIEW

**Comment by Keno** [comment]
LGTM. Thanks!

File: test/runtests.jl [hunk 1]
```diff
@@ -1,3 +1,14 @@
+using Base.Test
+
+# #316 test that import Cxx works without also using Cxx
+# this test must run before `using Cxx` is executed
+import Cxx
+@testset "import without using Cxx (#316)" begin
+    __current_compiler__ = Cxx.Cxx.__default_compiler__
```
Comment (vchuravy) [inline test/runtests.jl]: wrap this in a `let` block?

Comment (jw3126) [inline test/runtests.jl]: You mean replace the `@testset` by a let block? Why?

Comment (jw3126) [inline test/runtests.jl]: The `__current_compiler__` assignment does not leak out of the `@testset` into global scope, if that’s what you are concerned about.

**Comment by jw3126** [comment]
Are those CI fails expected, should I trigger a new run?

**Comment by Keno** [comment]
I've rebased this and the new CI should run properly. Will merge once that's through.

**Comment by jw3126** [comment]
Cool!

**Comment by ararslan** [comment]
Bump

**Comment by ararslan** [comment]
Any chance you could tag a new release now that this is fixed, @Keno?

**Pull request merged**